### PR TITLE
Feature/UI restyling v2

### PR DESCRIPTION
**Pull request by Steve-Rog**

### CODE REVIEW

**Comment by bachstatter** [comment]
Hey, I'm not sure this is ready for review, since it looks like there are some half finished things in there. Feel free to ping when ready. and maybe even keep it as a draft before.

Also not sure if you were planning to do this, but please provide some info in the description, including screenshot of the ui changes

**Comment by Steve-Rog** [comment]
> Hey, I'm not sure this is ready for review, since it looks like there are some half finished things in there. Feel free to ping when ready. and maybe even keep it as a draft before.
> 
> Also not sure if you were planning to do this, but please provide some info in the description, including screenshot of the ui changes

Hey. It's a rearrangement of the elements of the home page, and a redesign and of the header.  However, this change only makes sense when the other elements (Pnl Stats, OI Stats) are present, so maybe it can look "unfinished" for that reason.

As requested here is the screen:
screencapture-localhost-3000-2023-09-18-09_21_43

File: v2/perps-v2/ui/src/components/Dashboard/OpenInterests/OpenInterests.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { FlexProps } from '@chakra-ui/react';
+
+export const OpenInterests = ({ ...props }: FlexProps) => {
```
Comment (bachstatter) [inline v2/perps-v2/ui/src/components/Dashboard/OpenInterests/OpenInterests.tsx:3]: Is this WIP? Or can it be removed?

File: v2/perps-v2/ui/src/components/Dashboard/Pnl/Pnl.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+import { FlexProps } from '@chakra-ui/react';
+
+export const Pnl = ({ ...props }: FlexProps) => {
+  return (
+    <>
+    </>
+  );
+};
```
Comment (bachstatter) [inline v2/perps-v2/ui/src/components/Dashboard/Pnl/Pnl.tsx:9]: Same thing here

Comment (Steve-Rog) [reply v2/perps-v2/ui/src/components/Dashboard/Pnl/Pnl.tsx:9]: they can both be removed, I included them to facilitate this merge but actually I don't think they are needed

**Review by jmzwar — COMMENTED** [review]
Looks good @Steve-Rog , please remove what was already mentioned and we can gtm

**Comment by Steve-Rog** [comment]
> Looks good @Steve-Rog , please remove what was already mentioned and we can gtm



> Looks good @Steve-Rog , please remove what was already mentioned and we can gtm

changes committed

**Comment by jmzwar** [comment]
@Steve-Rog please ensure CI is passing before marking as ready to review.

**Review by jmzwar — APPROVED** [review]

**Pull request merged**